### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2371] [Q&A] [answered]
I don't know why but on my server group default have permission to heal tshock.heal i m tried to remove that permission but it is still there i even tried to remove it from all groups but the permission is still there! Fix your tshock please

**Comment by Ijwu** [comment, marked as answer]
I get you're frustrated. You haven't really explained to us what the issue is beyond "I tried removing permission and failed".

What is happening to indicate to you that the "permission s still there"? What do you expect to happen?

**Reply by ghost** [reply]
the permission is still there what hard to understand when i remove permission from Default group it is still there.

**Reply by Ijwu** [reply]
You've not really gone much into what you've done to try to remove the permission. How can anyone offer support if the process you've attempted is unclear? 

Again, I understand you're frustrated but if that frustration rears its head as hostility again I am closing this discussion permanently.
